src/main/java/com/amplitude/HttpTransport.java

```diff
@@ -5,11 +5,7 @@
 import org.json.JSONObject;
 
 import java.util.*;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.CompletionException;
-import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ConcurrentLinkedQueue;
-import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.*;
```

Comment:
Dose import java.util.* include java.util.concurrent.*?

Reply:
No, they are two packages. import java.util.* won't import classes in sub-packages.